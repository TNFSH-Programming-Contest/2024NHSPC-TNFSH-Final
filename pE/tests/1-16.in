1862 3267150 7790509 8547707
8870769 689744 805556
85 9396 537964 77 8410 823398 70 8070 912206 15 1742 678100 48 4403 833333 93 9301 929282 19 8180 733986 98 4122 538215 8 3007 848932 25 8458 780294 56 3558 654546 23 1449 944025 60 6441 921209 36 9939 802436 27 5699 562167 91 2040 544923 66 1321 897574 73 7913 675301 74 6487 971958 84 4346 647873 72 3998 966249 85 8535 786173 73 6300 533047 96 4840 606003 72 8732 955831 6 7233 851708 45 8272 639863 19 2061 728744 95 9938 572241 60 1576 774889 64 4667 703691 71 3447 991667 67 5888 891328 54 5162 948324 58 6600 945881 5 9230 803856 59 9731 947416 52 7437 950812 36 2875 704314 47 9299 782369 41 1744 898950 91 7284 794201 17 9343 796742 2 1839 669000 79 1091 597004 39 9407 676772 67 5458 685628 76 5258 772294 93 7151 932373 94 3934 575857 49 2211 745969 53 3685 528481 44 7341 908173 16 1233 756301 1 7919 559888 68 4758 921769 66 5749 693800 61 9644 899164 57 4990 793368 40 6646 697792 78 2640 787698 87 8153 981506 52 1730 619579 58 8789 913024 8 4577 688197 47 7696 725262 44 1679 609293 67 2973 534948 99 3988 753800 82 1058 997343 32 7326 997919 81 9311 757087 92 6841 795855 43 2051 769579 65 3481 657076 13 9027 680330 92 3045 652491 17 3977 780072 78 2227 548437 76 9682 624281 11 2415 770420 27 5331 901767 3 1496 890961 21 7279 864046 45 3163 557577 47 5790 904508 8 1053 726337 11 5011 768456 3 4701 843753 44 7539 657402 40 3045 564145 14 7429 812469 73 5992 559927 30 8416 672848 58 3501 755446 80 9757 757176 27 8462 531198 24 8056 511221 21 2175 915313 30 2981 725268 90 7696 817626 25 5155 888212 21 7467 611908 27 7892 864124 42 1295 707771 54 2333 837219 43 8607 898078 85 5087 629358 88 3806 669680 100 2742 636764 60 5345 900373 21 1469 836880 47 9516 842304 53 1473 865928 18 3277 744639 82 3078 919239 29 3237 917133 10 2152 646950 20 6265 727182 47 3555 860186 26 4863 940188 12 1477 739557 85 4487 846482 96 9971 731535 89 4591 741072 5 4117 956573 35 6784 514498 99 2957 759907 84 6001 511149 66 1632 804993 7 7103 599123 33 3056 727918 75 8420 785594 72 2119 665150 41 1943 927743 17 4942 943628 82 2367 627476 14 9016 687630 23 7939 983307 65 7991 734142 17 3584 555374 3 8829 606650 67 8677 610715 94 3664 842973 61 8334 572503 35 1801 822956 24 8819 583349 55 4000 650222 87 6965 974953 57 5721 628597 28 3358 971198 91 6490 883431 18 4861 609044 52 6358 724082 29 9438 664355 62 2902 964477 12 7510 795459 88 2606 536599 57 5768 766970 22 8203 970061 47 5021 833638 52 7752 715961 44 9308 808383 12 6532 960719 46 2157 844259 32 4464 597554 64 3610 694265 35 6473 524452 56 5341 646083 78 2696 864111 30 9917 976754 46 7792 578160 81 9892 652122 31 7457 643297 81 7104 878475 16 9312 867699 58 9849 746882 85 2769 742133 8 1991 685973 49 4518 799735 67 1918 920308 52 1544 892932 5 8957 889794 36 1970 583751 78 4892 545483 12 8632 606144 86 9745 626639 52 8794 772857 93 2441 605706 24 3624 651202 6 9364 657447 22 5219 884393 97 5214 581784 94 4415 607413 71 1411 703025 8 1041 751710 43 1579 700163 19 3684 996529 64 4057 746913 5 7429 885188 24 2312 634930 90 1348 611770 22 7114 891212 58 1745 782905 47 4261 953820 59 8254 810391 45 7327 806081 73 6724 980855 13 6338 933780 75 3113 767194 94 3864 676112 94 1005 843117 2 3832 982455 92 8655 931856 65 5197 765220 76 2368 616028 95 1606 884989 96 9588 993487 50 9197 682626 19 6233 865900 4 3185 660097 75 7572 569593 53 8376 920548 2 4205 888104 89 8254 743810 40 4249 707110 5 7477 546410 60 6823 516587 15 5320 764437 79 4647 611789 95 3873 961408 44 6545 526299 82 8429 538203 22 7361 547687 89 6737 778762 91 1750 755868 39 2483 935083 94 5449 666445 19 6018 826564 39 4248 688764 26 9120 524643 61 1694 903717 19 3269 970640 72 4359 629639 91 4986 745369 100 3435 784755 14 4270 990841 17 4918 579418 23 3328 754304 41 5817 937433 12 6555 876773 70 5593 627234 13 8730 759676 80 6549 922554 59 9209 710608 54 4519 903584 55 3916 858830 91 4800 681910 13 7155 795702 64 9047 840023 61 1067 979707 59 6771 864104 96 7538 988325 67 4281 687426 67 9451 841441 31 8760 569120 83 4040 582647 48 1208 849234 10 4623 694753 15 4311 660771 100 2868 753836 81 2719 833452 53 4747 533687 40 3175 732143 90 6723 890539 6 8655 537042 78 1006 928854 89 8529 509319 35 9815 584510 70 7762 829786 52 5169 651961 97 4437 500475 33 1735 913123 53 4104 917932 21 7833 667826 96 9554 523356 99 6541 846715 90 1926 934999 100 2754 954924 58 3435 967821 23 8280 702443 68 8512 754625 85 8515 654505 24 9472 559540 83 1044 982427 59 5562 625005 50 1346 520803 28 1721 609741 54 8084 723201 65 7252 539178 91 4821 661300 47 4371 776435 27 7077 630778 24 6468 803439 55 7519 876851 11 1246 669776 83 9953 975087 9 3775 561080 95 6523 582883 45 7198 748731 9 5102 744825 28 7380 804695 77 3290 633594 38 6443 682715 51 4477 696666 29 9726 621666 19 4908 986041 28 2565 806915 73 5993 961858 54 1592 729877 2 4049 588927 93 7659 942565 51 5892 934797 96 7870 762031 56 9292 939830 36 4580 689126 80 5815 555663 78 1098 714587 59 9843 523524 57 7400 730500 48 7514 791617 15 9957 787533 34 5938 920449 96 1772 859003 40 7167 837240 78 2732 505018 71 4011 662026 30 9940 895728 70 9670 879310 51 6293 966561 75 3009 976181 26 8149 982120 66 4333 711262 61 8321 793573 7 4096 558090 64 8778 649668 68 3164 592285 36 2729 649258 43 6498 984035 69 4792 796940 97 7485 686669 7 8875 684445 59 9953 957647 4 7284 779052 85 6127 646259 37 9690 847680 89 7812 895955 9 8009 729188 8 2435 633875 70 4353 895675 31 2566 784404 94 3525 979748 80 3845 677092 73 1163 683287 79 5564 735568 61 7992 682929 39 8508 654020 77 7244 672059 13 6144 544598 78 4593 956276 32 8767 620944 30 5036 941598 29 8824 848269 83 6629 729532 52 6453 782636 91 5159 907472 20 3630 550016 89 8310 747307 11 2355 803383 36 7759 973345 49 4195 655651 10 3136 959006 31 7633 865803 64 3891 641055 47 4498 902371 96 6284 511942 6 7584 683986 66 7906 631797 28 2821 539631 46 9683 762825 74 6829 903776 82 5003 989944 85 3062 610665 80 3506 809492 90 5831 833755 83 2369 960002 50 4851 668742 82 4117 796717 9 4320 737560 44 6720 943279 69 4802 925648 54 4632 813605 76 3439 934164 21 8229 773528 69 1195 655229 12 1841 677533 98 7785 954351 98 9543 949707 54 9562 949805 51 9290 512909 61 4257 850292 44 1019 808593 66 9800 913712 64 2034 773710 12 3055 658105 29 8494 991450 64 9475 656813 52 4332 620997 77 5979 867470 11 8037 671692 89 4484 786041 96 6741 677824 22 5001 886619 6 8641 658017 45 8256 997999 54 5342 572408 75 1642 872531 29 7280 796470 16 1793 522078 55 6780 954939 1 7376 556542 15 1791 635711 100 9873 785141 54 3089 639757 82 4918 649843 93 1241 999666 71 1187 766560 26 2733 983181 45 5731 920432 76 1881 857328 16 6259 763170 81 1710 760740 92 8825 682208 85 7206 573205 69 9287 697540 69 1453 779787 79 9237 738529 35 4679 754874 48 3687 733757 64 6472 779241 86 9880 662478 25 5283 624682 44 2584 709370 77 1411 982657 37 4661 709515 67 9629 540571 51 3462 524838 68 8941 711629 66 5989 573230 96 5930 564011 72 1788 547373 29 4426 912676 15 6860 878534 19 4922 606086 16 3832 723328 34 1722 657927 50 6747 991577 72 7394 517047 68 5642 858225 83 2353 635438 99 6292 711753 71 8506 645811 91 6577 691444 59 9503 720279 18 2176 734690 2 7970 661156 22 5338 667894 55 3811 587031 8 1013 738088 60 9318 753449 98 3808 987983 60 5376 645221 86 7583 844388 71 7273 684763 34 8948 660432 91 1752 902869 10 4400 630893 100 7926 548954 84 2168 782868 51 7554 672731 19 7044 833760 90 7757 986833 91 7146 985370 91 2268 873958 55 2051 701632 35 4836 600861 80 2689 504950 14 5053 863251 33 8646 603236 56 1264 793426 71 6141 500994 73 8024 796957 14 4652 815048 98 4271 540581 9 6073 873552 62 2012 914456 11 5400 738634 78 8053 553507 38 6802 648334 69 4900 964203 67 3842 747832 58 6215 971859 54 1660 835138 9 3462 558986 95 2015 646192 77 5261 719604 62 8191 593406 38 3765 559287 49 1672 653552 97 9686 693823 26 4236 965815 53 2207 997031 74 5547 764711 22 8220 783321 31 8105 649108 5 8088 702639 41 8974 813202 59 9781 855942 69 1853 988716 43 2842 895964 30 8954 764837 4 3476 951085 16 2837 894332 92 6240 640607 97 5171 743902 17 2045 715313 57 1736 738160 49 3565 951103 81 3439 791674 44 5685 530405 87 8349 881290 28 5950 734085 29 7153 744333 45 8535 703378 84 1693 836128 65 4458 989031 7 3096 738776 79 2724 615198 40 2118 547636 40 6481 915664 62 3589 734240 46 2554 968838 47 7150 500406 20 5378 583759 75 2754 848540 89 9861 741380 74 2616 616411 32 5197 851895 94 3916 725020 23 1822 867620 24 2118 993316 72 9534 892988 37 1977 665424 73 1260 679337 15 3559 773138 22 9653 916673 41 6233 544373 4 5828 732349 35 1359 738738 52 1739 964839 34 1628 557739 81 4171 902795 63 1025 666462 98 5432 533475 37 1361 856788 79 3278 851851 13 6657 680167 83 7452 690475 89 2790 638633 97 9647 986015 93 3933 938021 11 9136 685698 93 8768 673814 15 2192 580285 47 1671 689472 7 4836 943582 47 8048 998880 16 2072 990513 63 9744 789819 67 2874 872126 41 1102 640001 56 2407 597233 18 2094 580664 65 3929 799958 6 5278 736801 25 6470 667642 89 3094 896315 17 7536 783370 80 8999 772917 12 6130 551116 99 9895 515669 11 2645 577197 32 2990 950396 43 5670 823416 85 8979 792558 61 2479 583382 67 7247 977681 22 8832 683713 36 4663 894026 44 5913 547074 74 8907 908515 28 7537 866426 23 5464 671187 69 2847 827992 43 1775 616852 29 1296 817776 96 2714 973690 43 5652 577739 1 6342 943503 100 8881 694129 46 4558
