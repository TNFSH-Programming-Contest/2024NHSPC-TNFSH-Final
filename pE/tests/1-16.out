1271363097722
